scout/server/blueprints/variants/views.py

```diff
@@ -322,19 +322,31 @@ def variant_update(institute_id, case_name, variant_id):
         store.update_acmg(institute_obj, case_obj, user_obj, link, variant_obj, new_acmg)
         flash("updated ACMG classification: {}".format(new_acmg), 'info')
 
-    if request.form.getlist('dismiss_variant'):
-        new_dismiss = request.form.getlist('dismiss_variant')
+    new_dismiss = request.form.getlist('dismiss_variant')
+    if new_dismiss:
         store.update_dismiss_variant(institute_obj, case_obj, user_obj, link, variant_obj,
                                      new_dismiss)
         if new_dismiss:
             flash("Dismissed variant: {}".format(new_dismiss), 'info')
 
-    if variant_obj.get('dismiss_variant') and not request.form.getlist('dismiss_variant'):
-        new_dismiss = request.form.getlist('dismiss_variant')
+    if variant_obj.get('dismiss_variant') and not new_dismiss:
         store.update_dismiss_variant(institute_obj, case_obj, user_obj, link, variant_obj,
                                      new_dismiss)
         flash("Reset variant dismissal: {}".format(variant_obj.get('dismiss_variant')), 'info')
 
+    mosaic_tags = request.form.getlist('mosaic_tags')
+    if mosaic_tags:
+        store.update_mosaic_tags(institute_obj, case_obj, user_obj, link, variant_obj,
+                                     mosaic_tags)
+        if new_dismiss:
+            flash("Added mosaic tags: {}".format(mosaic_tags), 'info')
+
+    if variant_obj.get('mosaic_tags') and not mosaic_tags:
```

Comment:
..and this little shortcut won't work gracefully with multiple save buttons. I'll get back with a fix..